src/datomic_rest_api/rest/gene.clj

```diff
@@ -2191,34 +2226,56 @@
                :in $ ?gene
                :where [?vh :variation.gene/gene ?gene]
                       [?var :variation/gene ?vh]
-                      (not [?var :variation/allele _])]
+                      (not [?var :variation/allele _])
+                      (not [?var :variation/phenotype _])]
              db (:db/id gene))
           (map #(process-variation (entity db %))))
-     :description "polymorphisms and natural variations contained in the strain"}))
+     :description "polymorphisms and natural variations currently with no associated phenotype"}))
+
+(defn- alleles-count [gene]
```

Comment:
Pair on this would be nice. There is actually a deeper issue hinted here with large tables where fetch all the data would take a long time. The counts is an indicator that there are other alleles (alleles_other field) and polymorphisms, that user can fetch that's not initially fetched with the widget. But how it's currently done is kinda a hack. I'd like to change it soon.